build/Build.cs

```diff
@@ -95,30 +95,30 @@ class Build : NukeBuild
         .DependsOn(Test)
         .Executes(() =>
         {
-            var inputFolder = OctopusClientFolder / "bin" / Configuration / "net452";
-            var outputFolder = OctopusClientFolder / "bin" / Configuration / "net452Merged";
-            EnsureExistingDirectory(outputFolder);
-
-            var assemblyPaths = inputFolder.GlobFiles("NewtonSoft.Json.dll", "Octodiff.exe", "Octopus.TinyTypes.dll", "Octopus.TinyTypes.Json.dll", "Octopus.TinyTypes.TypeConverters.dll");
-
-            var inputAssemblies = new List<string> { inputFolder / "Octopus.Client.dll" };
-            inputAssemblies.AddRange(assemblyPaths.Select(x => x.ToString()));
-
-            var repackSettings = new RepackOptions()
+            foreach (var target in new [] {"net452", "netstandard2.0"})
             {
-                OutputFile = outputFolder / "Octopus.Client.dll",
-                InputAssemblies = inputAssemblies.ToArray(),
-                Internalize = true,
-                Parallel = false,
-                XmlDocumentation = true,
-                SearchDirectories = new [] { inputFolder.ToString() }
-            };
-
-            new ILRepack(repackSettings).Repack();
-
-            DeleteDirectory(inputFolder);
-            MoveDirectory(outputFolder, inputFolder);
-    });
+                var inputFolder = OctopusClientFolder / "bin" / Configuration / target;
+                var outputFolder = OctopusClientFolder / "bin" / Configuration / $"{target}Merged";
+                EnsureExistingDirectory(outputFolder);
+
+                var assemblyPaths = inputFolder.GlobFiles("NewtonSoft.Json.dll", "Octodiff.*", "Octopus.TinyTypes.dll", "Octopus.TinyTypes.Json.dll", "Octopus.TinyTypes.TypeConverters.dll");
```

Comment:
"Octopus.TinyTypes.dll", "Octopus.TinyTypes.Json.dll", "Octopus.TinyTypes.TypeConverters.dll" shouldn't be added here.
They need to be in the nuspec though.

Reply:
```suggestion
                var assemblyPaths = inputFolder.GlobFiles("NewtonSoft.Json.dll", "Octodiff.*");
```